server/container_create_linux.go

```diff
@@ -686,6 +699,10 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID, contai
 		specgen.AddMount(mnt)
 	}
 
+	if privileged {
+		setOCIBindMountsPrivileged(&specgen)
```

Comment:
does it also override the mode for bind mounts that were configured as `ro`?  Probably doesn't matter as the privileged container can always remount it

Reply:
Those are added after that call, so we're good. Thanks for reviewing!